index.html

```diff
@@ -2,8 +2,8 @@
 <html lang="en">
 <head>
 	<meta charset="UTF-8">
-    <meta http-equiv="Content-Security-Policy" content="default-src https:">
-	<link rel="stylesheet" href="styles.css" />
+	<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
I added this meta tag to help the site respond better on mobile browsers.

Reply:
How did I forget this 🤦‍♂️